BraintreeDropIn.xcodeproj/xcshareddata/xcschemes/DropInDemo.xcscheme

```diff
@@ -15,7 +15,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "A5CE1D131D835832009BE61A"
-               BuildableName = "DropInDemo.app"
+               BuildableName = "Drop-in Demo.app"
```

Comment:
Did we need to change the name of the `.app` file? Wouldn't just changing the `PRODUCT_NAME` suffice?

Reply:
Yeah, I was wondering that too. All I changed was the `Display Name` field, and the Xcode changed the rest. So... 🤷